### DISCUSSION TITLE
fail to set camera background camera in ManimGL v1.6.1

### DISCUSSION THREAD

**Discussion by zjcanjux** [3b1b/manim#1826] [Q&A]
```python
class TextExample(Scene):
    CONFIG = {
        "camera_config": {
            "background_color": WHITE
        }
    }
    def construct(self):
           pass
```

but my mp4 is still black 
<img width="794" alt="截屏2022-06-06 20 58 00" src="https://user-images.githubusercontent.com/37158421/172165194-d8febacb-048a-4b44-82e4-0070e7c221c6.png">

**Comment by TonyCrane** [comment]
This is a bug. Now you are only expected to set the background color by modifying the config file or using the CLI flag `-c WHITE`.

For the reason, see #1381.
